src/db/seed.js

```diff
@@ -0,0 +1,68 @@
+// use the class here to refactor
+
+// importing dependencies - DO NOT REMOVE
+const mysql = require("mysql2");
+const department = require("./data/department");
+const employee = require("./data/employee");
+const role = require("./data/role");
+const { dbOptions } = require("../db/databaseConfiguration.js");
+
+// creating connection to database
+const db = mysql.createConnection(dbOptions);
+
+// creating values for department
+const departmentValues = department
+  .map((department) => `("${department.name}")`)
+  .join(",");
+
+// template string query for department table
+const departmentQuery = `INSERT INTO department(name) VALUE${departmentValues}`;
+
+db.query(departmentQuery, (err, result) => {
+  if (err) {
+    console.log(err);
+    return;
+  }
+
+  console.log("Successfully entered DEPARTMENTS into department table.");
+  //   db.end();
+});
+
+// creating values for employee
+const employeeValues = employee
+  .map(
+    (employee) =>
+      `('${employee.first_name}', '${employee.last_name}', '${employee.role_id}', ${employee.manager_id})`
+  )
+  .join(",");
+
+// template string query for employee table
+const employeeQuery = `INSERT INTO employee(first_name, last_name, role_id, manager_id) VALUE${employeeValues}`;
+
+db.query(employeeQuery, (err, result) => {
```

Comment:
Your employees need to be seeded last after roles, due to the foreign key relationship here.